### ISSUE TITLE
Testsuite error in windows subsystem for linux (ubuntu)

### ISSUE THREAD

**Issue by jakobj** [nest/nest-simulator#1048]
When trying to run the testsuite in WSL, the following error occurs directly in the beginning

```bash
$ make installcheck
[...]
[...]/share/nest/extras/do_tests.sh: 360: export: (x86)/Intel/iCLS: bad variable name
[...]
```
NEST version: 2.16

does anyone have experience with this?

**Comment by heplesser** [comment]
@jakobj @terhorstd This got automatically closed when I merged #1500. I re-open so one of you can explicitly confirm that #1500 actually fixed this issue.

**Comment by heplesser** [comment]
We do not support WSL at present, therefore closing.
